tutor/src/tours/add-homework.json

```diff
@@ -26,7 +26,7 @@
   "steps": [
     {
       "title": "First, select a chapter",
-      "body": "Select a chapter you want to assign, then scroll down and click “show problems.” Uncheck any sections you don’t want to cover."
+      "body": "Select a chapter you want to assign, then scroll down and click \"Show Problems.\" Uncheck any sections you don't want to cover."
     }
```

Comment:
Hm, i seem to recall the mockup had UTF curly quotes.  But this is probably fine doubt it'll be noticed